pandas/io/formats/style.py

```diff
@@ -922,14 +953,18 @@ def set_td_classes(self, classes: DataFrame) -> Styler:
         ...     columns=[["level0", "level0"], ["level1a", "level1b"]])
         >>> classes = pd.DataFrame(["min-val"], index=["a"],
         ...     columns=[["level0"],["level1a"]])
-        >>> df.style.set_td_classes(classes)
+        >>> df.style.set_td_classes(classes)  # doctest: +SKIP
+                            level0
+                level1a	    level1b
+        a	    1	        2
+        b	    3	        4
 
         Form of the output with new additional css classes,
 
         >>> df = pd.DataFrame([[1]])
         >>> css = pd.DataFrame([["other-class"]])
         >>> s = Styler(df, uuid="_", cell_ids=False).set_td_classes(css)
-        >>> s.hide_index().render()
+        >>> s.hide_index().render()  # doctest: +SKIP
```

Comment:
this string might be worth doctesting (unless this ruins the pretty print for the user and therefore degrades the visibility in the docs)